src/mscorlib/facade/mscorlib.csproj

```diff
@@ -56,12 +56,11 @@
   <!-- Output paths -->
   <PropertyGroup>
     <BaseIntermediateOutputPath>$(RootBinDir)\obj</BaseIntermediateOutputPath>
-    <!-- Note the trailing '\\' - they are needed so that genfacade does not fail due to an invalid path
-         In particular, the facadepath is sent in quotes.
-         And the IntermediateOutputPath is required to end in a '\' in the coreclr repo. This means that the
-         facadepath ends up being sent as 'mypath\"' and that trips the Path normalization.
-         Introducing a second '\' tricks the normalization logic to treat this '\\"' as '\"' and not complain.    -->
-    <IntermediateOutputPath>$(BaseIntermediateOutputPath)\$(BuildOS).$(BuildArch).$(Configuration)\facade\\</IntermediateOutputPath>
+    <!-- Note the trailing '/' - it is needed so that genfacade does not fail due to an unmatched quote in the 
```

Comment:
I know this is ugly, but I haven't found any other way to get rid of the trailing backslash that kills the GenFacades.exe tool.

Reply:
This is fine.